lib/auth/auth.go

```diff
@@ -116,6 +116,13 @@ const (
 	mfaDeviceNameMaxLen = 30
 )
 
+const (
+	OSSDesktopsAlertId      = "oss-desktops"
+	OSSDesktopsAlertMessage = "OSS license only allows 3 Non-AD desktops configured. " +
```

Comment:
Let's get @xinding33 or @roraback to validate the copy here since this is a user facing CTA. 

I think we should be saying "Teleport Community Edition" instead of "OSS" but they can confirm.